keycloak/themes/hmda/login/template.ftl

```diff
@@ -83,6 +99,29 @@
       </div>
     </div>
   </div>
+
+  <footer class="usa-footer usa-footer-slim" role="contentinfo">
+    <div class="usa-grid usa-footer-return-to-top">
+      <a href="#">Return to top</a>
+    </div>
+    <div class="usa-footer-primary-section">
+      <div class="usa-grid-full">
+        <nav class="usa-footer-nav usa-width-one-half">
+          <ul class="usa-unstyled-list">
+            <li class="usa-footer-primary-content">
+              <a class="usa-footer-primary-link" href="https://www.ffiec.gov/">FFIEC</a>
+            </li>
+          </ul>
+        </nav>
+        <div class="usa-width-one-half">
+          <div class="usa-footer-primary-content usa-footer-contact_info">
+            <h4>Questions?</h4>
+            <a href="mailto:hmdahelp@cfpb.gov">hmdahelp@cfpb.gov</a>
```

Comment:
Can we use the configurable `${properties.supportEmailTo}` like we do [here](https://github.com/cfpb/hmda-platform-auth/pull/66/files#diff-04647443b51313258163baa6ea539479R83)?